qiskit/synthesis/__init__.py

```diff
@@ -77,8 +79,12 @@
     QDrift,
 )
 
+from .permutation import (
+    synth_permutation_depth_lnn_kms,
+    synth_permutation_basic,
+    synth_permutation_acg,
```

Comment:
should we call it `synth_permutation_depth_acg` to emphasize that we optimize depth?

Reply:
Thanks! I am afraid that we won't be able to find the perfect naming conventions. I am slightly in favor of keeping the current (shorter) variant of the name as there is only one "ACG" synthesis algorithm for permutations (the Alon, Chung, Graham method). In addition, while the ACG method does indeed guarantee the worst-case depth of 2 (measured in terms of SWAPs), it's also good for count, that's another reason why it might be better not to add "depth" to the name.